.github/actions/release/action.yaml

```diff
@@ -42,7 +42,7 @@ runs:
       env:
         GH_TOKEN: ${{ inputs.gh_token }}
         GITHUB_TOKEN: ${{ inputs.gh_token }}
-      run: npx nx affected --base=last-release --parallel=1 --target=version --postTargets=syncDependencies,npm,github --trackDeps
+      run: npx nx affected --base=last-release --parallel=1 --target=version --postTargets=build,syncDependencies,npm,github --trackDeps
```

Comment:
Since `ngx-deploy-npm` is dropping their internal build to use `dependsOn` Nx Feature and `postTarget` will not execute the target's dependencies, we need to specify ngx-deploy-npm dependencies here.

See more info on [Semver's Documentation](https://github.com/jscutlery/semver?tab=readme-ov-file#triggering-executors-post-release)